ios/FastImage/FFFastImageViewManager.m

```diff
@@ -34,5 +35,14 @@ - (FFFastImageView*)view {
     [[SDWebImagePrefetcher sharedImagePrefetcher] prefetchURLs:urls];
 }
 
-@end
+RCT_EXPORT_METHOD(clearMemoryCache)
+{
+    [SDImageCache.sharedImageCache clearMemory];
+}
 
+RCT_EXPORT_METHOD(clearDiskCache)
+{
+    [SDImageCache.sharedImageCache clearDiskOnCompletion:^(){}];
```

Comment:
Since this is asynchronous this method should return a `Promise` on the JS side. We don't want to expose this in a way where the user does not know if it's done or not.

Reply:
Fix that using promise api as below and I also rebase the code to master and also fix circleci error
https://github.com/DylanVann/react-native-fast-image/pull/425/files#diff-faa85178c446b76306807ed64ef30e39R59-R85
https://github.com/DylanVann/react-native-fast-image/pull/425/files#diff-cedfee9b06e731d51f287118e8a3ddb0R38-R47